changes/change_consumable_resource_checking.md

```diff
@@ -0,0 +1 @@
+Check whether each consumable resource is required as input from the submitter
```

Comment:
Is this changelog accurate at this point? I'm not sure I understand how these changes accomplish this change